examples/utils.py

```diff
@@ -7,7 +7,12 @@
 
 import colorama  # type: ignore
 
-from yapapi import windows_event_loop_fix, NoPaymentAccountError
+from yapapi import (
+    Golem,
+    windows_event_loop_fix,
+    NoPaymentAccountError,
+    __version__ as yapapi_version,
```

Comment:
This import is unused.

Reply:
which import?

Reply:
My bad, got confused by the name assigned through `as`.